downloads/swan-lake-release-notes/swan-lake-preview8/RELEASE_NOTE.md

```diff
@@ -0,0 +1,519 @@
+
+```ballerina
+transaction {
+    var res1 = client->callMyFirstService();
+   var  res2 = client ->callMySecondService();
+   var x = commit;
+    if (x is error) {
+        // error code
+    } else {
+        // success code
+    }
+}
+```                                                             
+
+#### Standard Library
+
+##### HTTP Module Changes
+
+###### Service Declaration
+
+- Basepath is removed from the `ServiceConfig`. Use the absolute resource path, which begins with `/` as the basePath. The `absolute-resource-path` is optional, which defaults to `/` when not specified.
+- The service type can be added after the `service` keyword as `http:Service`, which is also optional.
+
+**Old Syntax**
+
+```ballerina
+@http:ServiceConfig {
+    basePath: “hello”
+}
+service myService on new http:Listener(9090 {
+
+}
+```
+
+**New Syntax**
+
+```ballerina
+service http:Service /hello on new http:Listener(9090 {
+
+}
+```
+
+###### Resource Function Declaration
+
+- Use the accessor name to specify the HTTP method instead of the `methods` field of the `ResourceConfig` (e.g., `get`).
+- Use `default` the as accessor name when all standard HTTP methods need to be supported (e.g., the passthrough/proxy use case).
+- The resource path segement represents the `path`. The `path` field of the `ResourceConfig` is removed.
+- Use `.` to specify the resource path segement if the path needs to be set as `/`.
+- Path params are specified in the resource path segement within square brackets along with the type. The supported types are string, int, float, boolean, decimal (e.g., `path/[string foo]`).
+- Resource signature params are optional. Even the `Caller` and `Request` are optional and not ordered.
+- Query param binding support is added. The supported types are string, int, float, boolean, decimal, and the array type of them. The `Query` param type can be nialble (e.g., `(string? bar)`).
+- Rest param support is added. It can be used as a wildcard path segment to get the multiple path requests dispatched. Earlier it was used as `/*` and now it can be specified as `[string… s]` in which `s` is accessible within the resource. 
+- Data binding and resource function return support is not added yet.
+
+```ballerina
+service http:Service /mytest on helloEP {
+    resource function get  [string... extra](string? bar, http:Caller caller) {
+        // [int id] is a path param
+        // [string... extra] is a rest param
+        // string? bar is a query param
+    }
+}
+```
+
+###### Service Values Declaration
+
+```ballerina
+http:Service listenerMock = service object {
+    resource function get .(http:Caller caller, http:Request req) {
+        //...
+    }
+};
+
+err? e =  listenerEP.attach(listenerMock, "mock1"); // mock1 is the absolute-resource-path
+```
+
+##### Log Module Changes
+
+- Log levels are reduced to `INFO` and `ERROR`. There will be no user configuration to control the log level. All the logs are printed as standard errors.
```

Comment:
Are we trying to say that they're printed to the standard error stream?

Reply:
@MadhukaHarith92

Reply:
Yes.

Reply:
```suggestion
- Log levels are reduced to `INFO` and `ERROR`. There will be no user configuration to control the log level. All the logs will be printed to the standard error stream.
```